coderd/database/dbauthz/dbauthz_test.go

```diff
@@ -985,6 +985,16 @@ func (s *MethodTestSuite) TestOrganization() {
 			mem, policy.ActionRead,
 		)
 	}))
+	s.Run("PaginatedOrganizationMembers", s.Subtest(func(db database.Store, check *expects) {
+		o := dbgen.Organization(s.T(), db, database.Organization{})
+
+		check.Args(database.PaginatedOrganizationMembersParams{
+			OrganizationID: o.ID,
+			LimitOpt:       1,
```

Comment:
Can you add back the org member in the setup? And check it in the `Returns`? And set the limit to 0. 

That was breaking in the `dbmem` because `limit = 0` was not being treated as "no limit"